Classes/Projection/ProjectionManager.php

```diff
@@ -124,9 +125,15 @@ public function replay(string $projectionIdentifier)
 
         $filter = new EventTypesFilter($projection->getEventTypes());
 
-        foreach ($this->eventStore->get($filter) as $index => $eventWithMetadata) {
-            $listener = $this->eventListenerLocator->getListener($eventWithMetadata->getEvent(), $projection->getProjectorClassName());
-            call_user_func($listener, $eventWithMetadata->getEvent(), $eventWithMetadata->getMetadata());
+        try {
+            $eventStream = $this->eventStore->get($filter);
+        } catch (EventStreamNotFoundException $exception) {
+            return 0;
```

Comment:
Not directly related: Don't throw an exception when replaying a projection for an empty stream